includes/wikia/services/sass/Filter/CdnRewriteFilter.class.php

```diff
@@ -20,14 +20,14 @@ public function __construct( $cdnUrl ) {
 	public function process( $contents ) {
 		wfProfileIn(__METHOD__);
 
-		if (strpos($contents, "wgCdnStylePath") !== false) { // faster to skip the regex in most cases
+		if (strpos($contents, "wgResourceBasePath") !== false) { // faster to skip the regex in most cases
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app